types/nightwatch/index.d.ts

```diff
@@ -543,6 +543,7 @@ export interface NightwatchOptions {
 }
 
 export interface NightwatchGlobals {
+    [key: string]: any;
```

Comment:
This looks ok, I just want to highlight that the `interface NightwatchGlobals` can also be extended by the user like this:
```ts
declare module "nightwatch" {
    export interface NightwatchGlobals {
        foo: string;
        bar: number;
    }
}
```

Reply:
Thanks for highlighting 👍🏻, so my main reason for adding this is to reduce the problems faced by the developers (especially those who are new to typescript). This way if the choose to work with Nightwatch, then they don't face error if they try to use user-defined globals.

PS: I feel, having control over the types would be much better for bug-free codebase.